Mage/src/main/java/mage/abilities/effects/common/ExileThenReturnTargetEffect.java

```diff
@@ -91,6 +108,9 @@ public String getText(Mode mode) {
             sb.append(this.yourControl ? "your" : "its owner's");
         }
         sb.append(" control");
+        if (afterEffect != null){
+            sb.append(". ").append(afterEffect.getText(null));
```

Comment:
hmmm, not a huge fan of `getText(null)`... is the text wrong if you use `getText(mode)`? (but if you could in fact use a common condition and effect, it might be cleanest to also pass in a string for the conditional effect text anyway)